alpine-rails-buildkite/etc/sv/buildkite

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
+
+export HOME=/opt/buildkite
+export GEM_HOME=${HOME}/.gem
+
+cd $HOME
```

Comment:
I do not know yet. I just blindly reproduced the environment the official buildkite agent produces.

Note that by default the working directory for *any* process is `/opt/rails` due: https://github.com/dkubb/dockerfiles/blob/master/alpine-rails-nginx/Dockerfile#L45 which should be changed to `/`.